### PR TITLE
application_template: Add element for upstream commits

### PR DESCRIPTION
**Pull request by manankalra**
Closes https://github.com/coala/projects/issues/469

### CODE REVIEW

**Comment by manankalra** [comment]
Sorry about that.
I used a markdown editor - https://s9.postimg.org/qp5busmnz/random.png.
How do I render it locally?

**Comment by manankalra** [comment]
There's only one indentation level, that is for the `For example: ...` section.
Right, that is meant for both of them. I'll move it below the upstream part.

File: _faq/application_template.md [hunk 1]
```diff
@@ -23,13 +23,16 @@ Student Info
 Code Sample
 -----------
 
-Link all your contributions to coala and any other open source project here.
+- Link all your contributions to coala here.
```
Comment (jayvdb) [inline _faq/application_template.md]: unbullet this line, and then I think it is good enough for now.

File: _faq/application_template.md [hunk 2]
```diff
@@ -23,13 +23,16 @@ Student Info
...
+
 For example, to list your contributions to coala you can mention:
+  - https://github.com/coala/coala/commits/master?author=YOUR_USER_NAME
+    for coala repo
+  - https://github.com/coala/coala-bears/commits/master?author=YOUR_USER_NAME
+    for coala-bears repo
```
Comment (manankalra) [inline _faq/application_template.md]: In that case, should I remove indentation here?

File: _faq/application_template.md [hunk 3]
```diff
@@ -23,13 +23,16 @@ Student Info
...
 
-- https://github.com/coala/coala/commits/master?author=YOUR_USER_NAME
-  for coala repo
-- https://github.com/coala/coala-bears/commits/master?author=YOUR_USER_NAME
-  for coala-bears repo
+- Link all your contributions to other open source projects and upstream repositories
```
Comment (manankalra) [inline _faq/application_template.md]: and this bullet too? Looks odd with a single bullet in the middle of a section.

File: _faq/application_template.md (hunk 2 above)
Comment (jayvdb) [reply _faq/application_template.md]: No, I said "and then I think it is good enough for now."  that means the rest is fine.

File: _faq/application_template.md [hunk 4]
```diff
@@ -23,13 +23,16 @@ Student Info
 Code Sample
 -----------
 
-Link all your contributions to coala and any other open source project here.
+Link all your contributions to coala here.
```
Comment (jayvdb) [inline _faq/application_template.md]: please dont modify this line.

**Comment by jayvdb** [comment]
ack https://github.com/coala/projects/pull/524/commits/ae34d125e8ca133623dd50ce103aae24e7e1b76b

**Comment by jayvdb** [comment]
Please do a rebase according to our git guidelines

**Comment by jayvdb** [comment]
@gitmate-bot rebase

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**